apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -80,7 +80,10 @@ const triggerWebhook = async ({
       ...evt,
       downloadLink,
     }).catch((e) => {
-      console.error(`Error executing webhook for event: ${eventTrigger}, URL: ${webhook.subscriberUrl}`, e);
+      console.error(
+        `Error executing webhook for event: ${eventTrigger}, URL: ${webhook.subscriberUrl}, bookingId: ${evt.bookingId}, bookingUid: ${evt.uid}`,
+        e
+      );
```

Comment:
@Shpadoinkle can you please replace this with logger?

Reply:
you can import logger from "@calcom/lib/logger";

Reply:
@Udit-takkar   @keithwillcode     

updated